data-serving/data-service/test/util/search.test.ts

```diff
@@ -72,7 +74,7 @@ describe('search query', () => {
                 },
                 {
                     path: '_id',
-                    values: ['abc123'],
+                    values: [new ObjectId('605c8f6a7ee6c2d7fd2670cc')],
```

Comment:
This was a change I made to get the tests to pass. The fact that it _wasn't_ already part of the diff makes me suspect this test was just broken.